Source/Applications/SystemCenter/Model/Security/SecurityGroup.cs

```diff
@@ -278,7 +283,32 @@ protected override DataTable GetSearchResults(PostData postData, int? page)
                 row["DisplayName"] = group.DisplayName;
             }
 
-            // #ToDo Add Filtering by DisplayName and Type
+            DataTable filteredDataTable = dataTable.Clone();
```

Comment:
We might be able to do away with this variable entirely. Would it make sense to do .Clear() later instead and add the rows back into dataTable?

Reply:
CopyToDataTable seems to overwrite the existing rows so we don't have to .Clear it